routing/control_tower.go

```diff
@@ -264,8 +293,9 @@ func (p *controlTower) notifySubscribers(paymentHash lntypes.Hash,
 
 	// Get all subscribers for this payment.
 	p.subscribersMtx.Lock()
-	list, ok := p.subscribers[paymentHash]
-	if !ok {
+	subscribersAllPayments := p.subscribersAllPayments
```

Comment:
I don't think there's a need to add a new reference to the struct variable to save 2 characters... Just makes things harder to read. So should use `p.subscribersAllPayments` everywhere in this method.

Reply:
Done